packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -290,6 +290,7 @@ function BlockSelectionButton( { clientId, rootClientId } ) {
 						<BlockTitle
 							clientId={ clientId }
 							maximumLength={ 35 }
+							context="list-view"
```

Comment:
I don't think this will do anything since the BlockSelectionButton is no longer used in zoomed out mode since https://github.com/WordPress/gutenberg/pull/56808